layouts/partials/deadlines.html

```diff
@@ -7,7 +7,9 @@
 		<h2>{{ replace $translation.register.dates__heading "%state_name%" .Params.state_name }}</h2>
 	</header>
 {{end}}
-
+{{ if eq .Params.registration_type "not-needed"}}
+			{{/* Exclude registered-resources content for accessability*/}}
+	{{ else }}
```

Comment:
Can we replace these 3 lines with?
`{{ if ne .Params.registration_type "not-needed"}}`
It's simpler and doesn't require an empty return value.

Reply:
update made... that is a better path and not have just an empty thing